tutorial/markdown/java/spring-boot.md

```diff
@@ -22,29 +22,67 @@ sdk_language:
 length: 30 Mins
 ---
 
-<!-- 
+<!--
   The name of this file does not need to be `tutorial-quickstart-java-springboot` because it is in the `tutorials/java/markdown` directory, so we can just call it `spring-boot`. The idea is that we can leave off `tutorial-quickstart` as a prefix.
 -->
 
 <!-- TODO:  Figure out how to add width to image size in try it now links -->
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=java-springboot-quickstart-repo&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/java-springboot-quickstart)
 
-## Prerequisites
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Source Code](#source-code)
+  - [Install Dependencies](#install-dependencies)
+  - [Database Server Configuration](#database-server-configuration)
+  - [Application Properties](#application-properties)
+- [Running The Application](#running-the-application)
+  - [Directly on the machine](#directly-on-the-machine)
+  - [Docker](#docker)
+- [What We'll Cover](#what-well-cover)
+  - [Document Structure](#document-structure)
+  - [Code Organization](#code-organization)
+- [Data Model](#data-model)
+- [Let's Review the Code](#lets-review-the-code)
+  - [Model](#model)
+  - [Controller](#controller)
+  - [Service](#service)
+  - [Repository](#repository)
+- [Mapping Workflow](#mapping-workflow)
+  - [GET Mapping Workflow](#get-mapping-workflow)
+  - [POST Mapping Workflow](#post-mapping-workflow)
+  - [PUT Mapping Workflow](#put-mapping-workflow)
+  - [DELETE Mapping Workflow](#delete-mapping-workflow)
+- [Custom SQL++ Queries](#custom-sql-queries)
+- [Running The Tests](#running-the-tests)
+  - [Run Individual Tests](#run-individual-tests)
+- [Project Setup Notes](#project-setup-notes)
+- [Contributing](#contributing)
+- [Appendix](#appendix)
+  - [Extending API by Adding New Entity](#extending-api-by-adding-new-entity)
+  - [Running Self Managed Couchbase Cluster](#running-self-managed-couchbase-cluster)
+  - [Swagger Documentation](#swagger-documentation)
+
+## Getting Started
+
+### Prerequisites
 
 To run this prebuilt project, you will need:
 
-- Follow [Couchbase Installation Options](/tutorial-couchbase-installation-options) for installing the latest Couchbase Database Server Instance (at least Couchbase Server 7)
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/dotnet-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+  - To run this tutorial using a self managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
 - Java SDK v1.8 or higher installed
```

Comment:
Java SDK 1.8 seems too old.

Reply:
made 17

Reply:
??

Reply:
<img width="1109" alt="image" src="https://github.com/couchbase-examples/couchbase-tutorials/assets/44238657/097adb66-4a76-4bb6-b795-e9420f24bc4b">

Github is showing the outdated one